deposit/src/main/java/edu/unc/lib/deposit/normalize/DirectoryToBagJob.java

```diff
@@ -113,10 +117,19 @@ public void runJob() {
 				
 				try {
 					checksum = DigestUtils.md5Hex(new FileInputStream(fullPath));
+					
+					PremisLogger premisDepositLogger = getPremisLogger(getDepositPID());
```

Comment:
Both of these events should be events on the object checksummed rather than the deposit